
# HG changeset patch
# User Torsten <ttl@justmail.de>
# Date 1449207825 -3600
# Node ID e870a68742a6c6e86333c72956aa31ceb7986131
# Parent  f7b0572fff6e5e555a3ecea2ee5c9bf8b6798910
change alignment of dock widget titles for OSX (bug #46592)

* octave-dock-widget.cc (set_style): change alignment in style sheet for OSX

diff --git a/libgui/src/octave-dock-widget.cc b/libgui/src/octave-dock-widget.cc
--- libgui/src/octave-dock-widget.cc
+++ libgui/src/octave-dock-widget.cc
@@ -330,6 +330,11 @@
   else
     dock_icon = "widget-undock";
 
+#if defined (Q_OS_MAC)
+  QString alignment = "center";
+#else
+  QString alignment = "center left";
+#endif
   if (_custom_style)
     {
 
@@ -371,7 +376,7 @@
       css = background + QString (" color: %1 ;").arg (fg_col.name ());
 #else
       css = QString ("QDockWidget::title { " + background +
-                     "                     text-align: center left;"
+                     "                     text-align: " + alignment + ";"
                      "                     padding: 0px 0px 0px 4px;}\n"
                      "QDockWidget { color: %1 ; "
                      "  titlebar-close-icon: url(:/actions/icons/widget-close%2.png);"
@@ -387,7 +392,7 @@
 #if defined (Q_OS_WIN32)
       css = QString ("");
 #else
-      css = QString ("QDockWidget::title { text-align: center left;"
+      css = QString ("QDockWidget::title { text-align: " + alignment + ";"
                      "                     padding: 0px 0px 0px 4px;}"
                      "QDockWidget {"
                      "  titlebar-close-icon: url(:/actions/icons/widget-close.png);"

